src/pages/elements/NewNotification.js

```diff
@@ -0,0 +1,520 @@
+import React, { Fragment } from 'react';
+import { Query } from 'react-apollo';
+import ApolloClient from 'apollo-boost';
+import gql from 'graphql-tag';
+
+import "@patternfly/react-core/dist/styles/base.css";
```

Comment:
This is surprising? Is it not loaded by default?

Reply:
No, it is not loaded by default. ;(